lib/rspec-rails.rb

```diff
@@ -5,11 +5,13 @@ module Rails
     # Railtie to hook into Rails.
     class Railtie < ::Rails::Railtie
       # Rails-3.0.1 requires config.app_generators instead of 3.0.0's config.generators
-      generators = config.respond_to?(:app_generators) ? config.app_generators : config.generators
+      generators = config.respond_to?(:app_generators) ?config.app_generators : config.generators
       generators.integration_tool :rspec
-      generators.test_framework   :rspec
+      generators
+      .test_framework   :rspec
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Unnecessary spacing detected.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.